### ISSUE TITLE
Coachmark: Color contrast ratio failed of Close button boundary under 'Example Title' pane.

### ISSUE THREAD

**Issue by dzearing** [OfficeDev/office-ui-fabric-react#11149]
Tracking all of the issues brought up with the 2/18 accessibility pass on Coachmark control
Details on individual bugs can be found on [Sharepoint](https://microsoft-my.sharepoint.com/:x:/r/personal/v-amaver_microsoft_com/_layouts/15/Doc.aspx?sourcedoc=%7Bec787de0-7a76-4b21-b593-fb839dfa9cd8%7D&action=default&uid=%7BEC787DE0-7A76-4B21-B593-FB839DFA9CD8%7D&ListItemId=1808&ListId=%7B395E1ECB-5002-440E-81C5-6A090E98077C%7D&odsp=1&env=prodbubble)
